scalableminds/webknossos
# Manual Trigger for Dataset directory Inbox Check

(master-based recreation of #2437)

### Mailable description of changes:
- Adds a button for immediately checking the binaryData folder for new datasets

### Steps to test:
- Change a dataset name in your binaryData folder
- Press Refresh in the Datasets view
- Renamed dataset should appear

### Issues:
- fixes #2434
- fixes #2444

------
